packages/web-pkg/src/components/AppBanner.vue

```diff
@@ -56,14 +55,12 @@ export default defineComponent({
     const isVisible = ref<boolean>(unref(appBannerWasClosed) === null)
     const store = useStore()
     const router = useRouter()
-
     const appBannerSettings = unref(store.getters.configuration.currentTheme.appBanner)
-
-    const appUrl = computed(() =>
-      buildUrl(router, `/f/${props.fileId}`)
+    const appUrl = computed(() => {
+      return buildUrl(router, `/f/${props.fileId}`)
         .toString()
-        .replace(/^(http)(s)?/, (_, p1, p2) => (p2 ? appBannerSettings.appScheme : p1))
-    )
+        .replace('https', appBannerSettings.appScheme)
```

Comment:
Why did you switch from the regexp? We should not rely on the url using https imho

Reply:
But we're only supporting https, no? Or better not make an assumption here? 🤔

Reply:
https forever